apis/swagger.yml

```diff
@@ -1060,6 +1060,14 @@ definitions:
                 type: "object"
                 additionalProperties:
                   type: "string"
+          RestartPolicy:
```

Comment:
You can use:
```
RestartPolicy:
    $ref: "#/definitions/RestartPolicy"
```

and in definitions, you can define `RestartPolicy`:
```
RestartPolicy:
     type: "object"
     description: "Restart policy to be used to manage the container"
     properties:
          Name:
              type: "string"
          MaximumRetryCount:
              type: "integer"
```

Then you can get a type of RestartPolicy instead of `HostConfigAO0RestartPolicy `.

Reply:
ok